tools/configen/configen/configen.py

```diff
@@ -132,11 +133,33 @@ def generate_module(cfg: ConfigenConf, module: ModuleConf) -> str:
     classes_map: Dict[str, ClassInfo] = {}
     imports = set()
     string_imports: Set[str] = set()
+
```

Comment:
can you extract this logic into a function?

Reply:
which logic?

Reply:
The 20 or so new lines in this loop that are handling the flags.